adapter.go

```diff
@@ -39,13 +40,13 @@ type customTableKey struct{}
 
 type CasbinRule struct {
 	ID    uint   `gorm:"primaryKey;autoIncrement"`
-	Ptype string `gorm:"size:100;uniqueIndex:unique_index"`
-	V0    string `gorm:"size:100;uniqueIndex:unique_index"`
-	V1    string `gorm:"size:100;uniqueIndex:unique_index"`
-	V2    string `gorm:"size:100;uniqueIndex:unique_index"`
-	V3    string `gorm:"size:100;uniqueIndex:unique_index"`
-	V4    string `gorm:"size:100;uniqueIndex:unique_index"`
-	V5    string `gorm:"size:100;uniqueIndex:unique_index"`
+	Ptype string `gorm:"size:100"`
+	V0    string `gorm:"size:100"`
+	V1    string `gorm:"size:100"`
+	V2    string `gorm:"size:100"`
+	V3    string `gorm:"size:100"`
+	V4    string `gorm:"size:100"`
+	V5    string `gorm:"size:100"`
```

Comment:
Why remove this?

Reply:
he would like create unique index manually

Reply:
What's the benefit?

Reply:
for different adpater instance using the same table.
But I don't think it's necessary

Reply:
@yangzhares

Reply:
Assume we have a database named `casbin`, under this database, in current codes, we can call `NewAdapterByDBUseTableName` to successfully create an adapter, but if do a second call, that would get an error due to index name conflict, because current on `CasbinRule` struct unique index name already had been hard-code. 

So i want to remove unique index tag on `CasbinRule` struct, and manually creates unique index when calls `createTable` to create a table, this can make sure each table(adapter) has a different unique index name and avoid name conflict

Reply:
@closetool

Reply:
@yangzhares I saw duplicated code like: `tableName = a.tablePrefix + "_" + tableName`. Plz define `getFullTableName()`

Reply:
please review again